include/tester/db/LogMessage.hpp

```diff
@@ -2,21 +2,21 @@
 
 #include "LogSeverity.hpp"
 
-class MessageLogger
+class LogMessage
 {
 public:
-    MessageLogger()
+    LogMessage()
```

Comment:
Is this constructor necessary?
- If it is - use ` = default`
- If not - remove it.

Reply:
done